core/src/main/resources/lib/layout/pageHeader.jelly

```diff
@@ -25,82 +25,6 @@
-
-                    <span class="main-search__icon-leading">
-                        <l:svgIcon href="${resURL}/images/material-icons/svg-sprite-action-symbol.svg#ic_search_24px" />
-                    </span>
-                    <a href="${searchHelpUrl}" class="main-search__icon-trailing">
-                        <l:svgIcon href="${resURL}/images/material-icons/svg-sprite-action-symbol.svg#ic_help_outline_24px" />
-                    </a>
-
-                    <div id="search-box-completion" />
-                    <script>createSearchBox("${searchURL}");</script>
-                </div>
-            </form>
-        </div>
-
-        <div class="login page-header__hyperlinks">
-            <div id="visible-am-insertion" class="page-header__am-wrapper" />
-            <div id="visible-sec-am-insertion" class="page-header__am-wrapper" />
-
-            <!-- login field -->
-            <j:if test="${app.useSecurity}">
-                <j:choose>
-                    <j:when test="${!h.isAnonymous()}">
-                        <j:invokeStatic var="user" className="hudson.model.User" method="current" />
-                        <j:choose>
-                            <j:when test="${user.fullName == null || user.fullName.trim().isEmpty()}">
-                                <j:set var="userName" value="${user.id}"/>
-                            </j:when>
-                            <j:otherwise>
-                                <j:set var="userName" value="${user.fullName}"/>
-                            </j:otherwise>
-                        </j:choose>
-                        <a href="${rootURL}/${user.url}" class="model-link inside inverse">
-                            <l:svgIcon
-                                       class="am-monitor-icon" >
-                                <use href="${resURL}/images/material-icons/svg-sprite-social-symbol.svg#ic_person_24px"></use>
-                            </l:svgIcon>
-                            <span class="hidden-xs hidden-sm">${userName}</span>
-                        </a>
-                        <j:if test="${app.securityRealm.canLogOut()}">
-                            <a href="${rootURL}/logout">
-                                <l:svgIcon href="${resURL}/images/material-icons/svg-sprite-action-symbol.svg#ic_input_24px" />
-                                <span class="hidden-xs hidden-sm">${logout}</span>
-                            </a>
-                        </j:if>
-                    </j:when>
-                    <j:otherwise>
-                        <st:include it="${app.securityRealm}" page="loginLink.jelly" />
-                    </j:otherwise>
-                </j:choose>
-            </j:if>
-        </div>
-    </header>
+	<j:invokeStatic var="header" className="jenkins.views.Header" method="get"/>
+	<st:include it="${header}" page="headerContent.jelly"/>
```

Comment:
nit, rest of the file uses spaces

Reply:
Tabs removed in https://github.com/jenkinsci/jenkins/pull/5909/commits/68a6b9c2768fb5e328dfcdd1740959a617ffef69. Thanks for the heads up @timja!